### PR TITLE
changed Key component to make copying to clipboard visible

### PR DESCRIPTION
**Pull request by flipsimon**

### CODE REVIEW

**Comment by Frando** [comment]
Looks good!

File: sonar-ui/src/components/Key.js [hunk 1]
```diff
@@ -1,29 +1,75 @@
...
-  return (
-    <Text {...props} onClick={e => copyToClipboard(key)}>
-      {shortkey}
-    </Text>
-  )
+  const [copyValue, setCopyValue] = useState(key)
+  const { value, onCopy, hasCopied } = useClipboard(copyValue)
+  // var ttColor
+  if (!hasCopied) {
+    return (copyClipboardTooltip())
```
Comment (Frando) [inline sonar-ui/src/components/Key.js]: The brackets around the function call are not needed,

File: sonar-ui/src/components/Key.js [hunk 2]
```diff
@@ -1,29 +1,75 @@
...
+  }
+  return (copiedBadge())
```
Comment (Frando) [inline sonar-ui/src/components/Key.js]: This style is a bit unusual for React components, but also doesn't really matter much

File: sonar-ui/src/components/Key.js [hunk 3]
```diff
@@ -1,29 +1,75 @@
...
+
+  function copyClipboardTooltip () {
+    return (
+      <Tooltip
+        placement='right-end'
+        width='148px'
+        label={
+          <Flex direction='row' justify='space-between'>
+            <Box flex='1'><Icon name='copy' /></Box>
+            <Box flex='5'><Text> Copy to clipboard</Text></Box>
```
Comment (Frando) [inline sonar-ui/src/components/Key.js]: Extra space before Copy

File: sonar-ui/src/components/Key.js (hunk 2 above)
Comment (flipsimon) [reply sonar-ui/src/components/Key.js]: what would be better? this?
```suggestion
  return (hasCopied ? copiedBadge() : copyClipboardTooltip() )
```

**Pull request merged**